src/locale/i18n.ts

```diff
@@ -198,6 +199,14 @@ export async function dynamicActivate(locale: AppLanguage) {
       ])
       break
     }
+    case AppLanguage.ro: {
+      i18n.loadAndActivate({locale, messages: messagesRo})
+      await Promise.all([
+        import('@formatjs/intl-pluralrules/locale-data/ro'),
```

Comment:
```suggestion
        import('@formatjs/intl-datetimeformat/locale-data/ro'),
        import('@formatjs/intl-pluralrules/locale-data/ro'),
```

Following #6742 being merged, could you add this line please?

This will ensure that the date and time for posts are localized correctly on all platforms. Thanks!